src/main/java/org/gaul/s3proxy/Main.java

```diff
@@ -235,6 +246,15 @@ public static void main(String[] args) throws Exception {
         }
     }
 
+    private static PrintStream createLoggerErrorPrintStream() {
+        return new PrintStream(new LogOutputStream() {
+            @Override
+            protected void processLine(String s, int i) {
```

Comment:
Sorry, I will not accept cut and paste code.  Also you need to at least open an issue against args4j if you do not provide a fix.